gi_loadouts/face/wind/fclt.py

```diff
@@ -139,14 +139,16 @@ def weap_save(self) -> None:
                     refn=self.weap_area_refn.currentText(),
                 )
 
-                file.save(
+                save_status = file.save(
                     self,
                     "Select location to save weapon data",
                     f"{objc.name.strip().replace(" ", "").replace("\"", "").replace("-", "").replace("'", "")}_{uuid4().hex[0:8].upper()}_{datetime.now().strftime("%Y%m%d_%H%M%S")}",
                     objc,
                 )
-
-                show_status(self.statarea, "Weapon data has been successfully saved.")
+                if save_status:
+                    show_status(self.statarea, "Weapon data has been successfully saved.")
+                else:
+                    show_status(self.statarea, "Save process cancelled.")
```

Comment:
It is better to just add `return` as used in the code for loading artifacts and weapons (https://github.com/TAKI12T/gi-loadouts/blob/11337829e46ed61245fae9b0be8dcc4b32df47dc/gi_loadouts/face/wind/fclt.py#L173-L174). 

No need for an extra message.

```suggestion
                if not status:
                    return
                show_status(self.statarea, "Weapon data has been successfully saved.")
```